Gemfile

```diff
@@ -1,3 +1,9 @@
 source 'https://rubygems.org'
 
+group :development, :test do
+  gem 'simplecov', require: false
+  gem 'rubocop', '~> 0.37.2', require: false unless RUBY_VERSION =~ /^1.8/
```

Comment:
@mikz if we only run on code climate, does that mean it's added as a build status thing alongside Travis?